site/installing.md

```diff
@@ -169,13 +169,16 @@ To enable DaemonSets in an existing cluster, add the `--runtime-config=extension
 
 **Install Scope on Your Cluster**
 
-It is recommended that you run Scope natively in your Kubernetes cluster using the manifest generator service.
+It is recommended that you run Scope natively in your Kubernetes cluster using `kubectl` with launch URL as show below.
 
 The simplest way to get the latest release of Scope deployed onto a Kubernetes cluster is the following:
 
-    kubectl create -f 'https://scope.weave.works/launch/k8s/weavescope.json' --validate=false
+    kubectl --namespace=kube-system create -f 'https://scope.weave.works/launch/k8s/weavescope.json' --validate=false
```

Comment:
Also, I have to say, I did consider inferring the namespace already, before the OpenShift question came to mind. And actually the main argument was this.

```
kubectl create -f 'https://scope.weave.works/launch/k8s/weavescope.json'
```
Next, user will try running either `kubectl get pods` or `kubectl get evets -w` and they won't see anything about Scope. The user will come and ask us and we will have to remind them it's in `kube-system` namespace. It's much much better for this to be explicit, and also works well for OpenShift.